src/elements/content-sidebar/activity-feed/common/activity-message/messages.js

```diff
@@ -24,8 +24,9 @@ const messages = defineMessages({
     },
     activityMessageEdited: {
         id: 'be.contentSidebar.activityFeed.common.editedMessage',
-        defaultMessage: ' (edited)',
-        description: 'Label indicating that message was edited, should be lowercase',
+        defaultMessage: '\\ (edited)',
+        description:
+            'Label indicating that message was edited, should be lowercase, leading space should be escaped with \\',
```

Comment:
I've asked them

Reply:
That will work, yes, especially since you added the note in the description. I will check the translations to make sure that the escaping appears in the translations as well.